app/models/taxable_taxonomy.rb

```diff
@@ -6,7 +6,7 @@ class TaxableTaxonomy < ActiveRecord::Base
 
   scope :without, lambda{ |types|
     if types.empty?
```

Comment:
condition = types.present? ? ["taxable_taxonomies.taxable_type NOT IN (?)",types] : {}
where(condition)